neuro-cli/src/neuro_cli/topics.py

```diff
@@ -376,14 +376,14 @@ async def aliases() -> None:
     ------------
 
     In **pattern mode** the system command is used along with **substitutions**,
-    e.g. `exec = "du {human_readable} {max_depth} {file}"`.
+    e.g. `exec = "du {human_readable} {max_depth} {file}`.
     Substitution is a variable name to expand enclosed in figure brackets,
     e.g. `{file}`.
 
     It is expanded with an option or positional argument specified
     by `args` or `options`.  The substitution name is automatically lowercased,
     minus (`-`) is replaced with underscore (`_`).
-    E.g. `args = "ARG-NAME"` matches to `{arg_name}`.
+    E.g. `args = "ARG-NAME` matches to `{arg_name}`.
```

Comment:
Something wrong here

Reply:
Sorry, I was 100% sure I only committed related chunks and not these ones. My apologies for not double-checking this fact in the PR.